packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -882,10 +882,12 @@ class RawScrollbarState<T extends RawScrollbar> extends State<T> with TickerProv
   @override
   void didUpdateWidget(T oldWidget) {
     super.didUpdateWidget(oldWidget);
-    if (widget.isAlwaysShown != oldWidget.isAlwaysShown) {
+    // If `isAlwaysShown` is true and does not change,
+    // it may be necessary to trigger a scroll event to show or hide the bar when the
+    // scrollable widget viewport size changed.
+    if (widget.isAlwaysShown == true || widget.isAlwaysShown != oldWidget.isAlwaysShown) {
       if (widget.isAlwaysShown == true) {
         _maybeTriggerScrollbar();
-        _fadeoutAnimationController.animateTo(1.0);
```

Comment:
I remove this line because I think this is redundant and the scroll event will trigger the animate at line 1068,
```dart
      // Any movements always makes the scrollbar start showing up.
      if (_fadeoutAnimationController.status != AnimationStatus.forward
          && _fadeoutAnimationController.status != AnimationStatus.completed)
        _fadeoutAnimationController.forward();
```

Reply:
but it will not be shown before scroll happens. it should not be removed

Reply:
without scroll event, the bar still can not be painted.